.travis.yml

```diff
@@ -0,0 +1,20 @@
+# currently unused. retained for potential future use to validate docker image builds
+sudo: required
+services:
+  - docker
+node_js:
+  - 10.16.3
+before_install:
+  - node ./deploy/should-it-build.js $TRAVIS_PULL_REQUEST
+script:
+  - cd admin
+  - docker build .
+  - cd ../admin-assets
+  - ./copy-assets.sh
+  - docker build .
+  - cd ../pupil-spa
```

Comment:
These 3 tasks could be parallelised in travis
https://docs.travis-ci.com/user/speeding-up-the-build/

Reply:
nicely remembered 👌